src/style.css

```
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: flex;
+  margin: 0 auto;
+}
+
+.card__description {
+  margin: 40px 0 0;
+  font-style: normal;
+  line-height: 18px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__code {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 0 0;
```

Can you use margin-top?

```suggestion
  margin-top: 4px;
```